app/models/form_application.rb

```diff
@@ -0,0 +1,18 @@
+class FormApplication < ActiveRecord::Base
```

Comment:
What I meant is: the naming is confusing. I wouldn't be able to tell the semantic difference between the two models. However, let's roll with it and change it elsewhere. It's not a show-stopper :smile: